application/app/src/main/resources/application.yaml

```diff
@@ -0,0 +1,5 @@
+server:
+  port: 9095
+
+param:
+  version: "0.0.1"
```

Comment:
Can we store the API version in the Make profile?

Reply:
Yes.